### ISSUE TITLE
Rewrite rules are broken

### ISSUE THREAD

**Issue by trebitzki** [aymericb/exposition#2]
Please refer to issue #1 for my setup. The .htaccess line `RewriteBase /exposition` seems to cause or not prevent a rewrite to http://localhost/, so a browser refresh throws me out of the gallery.

**Comment by aymericb** [comment]
I think `RewriteBase /exposition` is not enough. Try also changing the line just below from `RewriteRule ^(.+)/?$ ?path=/$1 [L,R]` to `RewriteRule ^(.+)/exposition/?$ ?path=/$1 [L,R]` or something like that.

As I said, I do really support deploying to anything but '/' at the moment.

PS: Sorry I cannot test this now.

**Comment by trebitzki** [comment]
No, I tried some variations on that, same spiel. I'll have a look at your js code over the weekend; maybe I can come up with a patch.

**Comment by aymericb** [comment]
Oh yeah. It clashes with the HTML5 history rewrite...
